### DISCUSSION TITLE
To control third axis with h4

### DISCUSSION THREAD

**Discussion by tektonandroid** [kachurovskiy/nanoels#182] [Q&A]
Hello everyone. I would like to control a third axis. Specifically, it would be a Y axis of a vertical accessory that I would use for small milling. Maybe you could motorize it and control the height using the H4. It's possible? It's supposed to be able to control up to 4 axes, right?
Greetings.

**Comment by kachurovskiy** [comment]
Hi @tektonandroid, yes it's possible, please check out lines 45-56 to configure it. To move the axis you need to press the Mode button until you enter A1 mode. Then you can use the top row of buttons to move and set limits.

**Reply by tektonandroid** [reply]
When you indicate the top row of buttons, what buttons are you referring to?
Regards.

**Reply by kachurovskiy** [reply]
![Screenshot_20240223-182119](https://github.com/kachurovskiy/nanoels/assets/517919/82f9be81-9380-498d-a99d-e0701fb6ed52)

**Comment by 46m27** [comment]
buongiorno
ho costruito h4, davo dire che è veramente bello, il mio problema è che non riesco ad accedere ad A1 per collegare un motore, il tasto M 
mi fa accedere a tutte le funzioni escluso A1, come posso fare?
Saluti

**Reply by kachurovskiy** [reply]
Hi, you need to enable A1 in line 45 - https://github.com/kachurovskiy/nanoels/blob/main/h4/h4.ino#L45-L56

**Comment by 46m27** [comment]
grazie delle tempestiva risposta, ho visto la riga 45 (const bool ACTIVE_A1 = false; ) ma non sono capace ad attivare, come faccio?
scusa per il disturbo

**Comment by 46m27** [comment]
ci sono riuscito, grazie infinite del tuo aiuto
saluti
